tinker/static/assets/js/tinker.js

```diff
@@ -14,4 +14,117 @@ $(document).ready(function () {
             $(this).text("- Less Search Filters")
         }
     });
-});
\ No newline at end of file
+});
+
+function pagination() {
+    var numberOfItems = $(" #loop .list-group").length;
+    var limitPerPage = $("#first").val();
```

Comment:
that id seems vague?

Reply:
Changed to ```selected-default``` since it is the option that is set to selected to start.

Reply:
why not match the other id names? `10`. In which case could be cleaner to be `show-10`. And then you just parse the number. Or, you use html data attributes to hold the data

Reply:
So do something like this?

`var id = $("#selected-option").children(":selected").attr("id");`

Reply:
I am now using ids to get the values needed